packages/components-native/src/Button/Button.tsx

```diff
@@ -143,17 +167,36 @@ export function Button({
     >
       <View style={buttonStyle}>
         {loading && <InternalButtonLoading variation={variation} type={type} />}
-        <View style={getContentStyles(label, icon)}>
+        <View
+          style={getContentStyles(label, icon, UNSAFE_style?.contentContainer)}
+          testID="contentContainer"
+        >
```

Comment:
We can also remove the logic added below in `getContentStyles` and revert this back to

```suggestion
<View
          style={[
            getContentStyles(label, icon),
            UNSAFE_style?.contentContainer,
          ]}
          testID="contentContainer"
        >
```